front/src/components/Header.jsx

```diff
@@ -68,24 +67,7 @@ export const Header = () => {
               </>
             ) : (
               <>
-                <>
-                  <Link to="/login">
-                    <button
-                      className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none dark:focus:ring-blue-800"
-                      variant="outlined"
-                    >
-                      Login
-                    </button>
-                  </Link>
-                  <Link to="/register">
-                    <button
-                      className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none dark:focus:ring-blue-800"
-                      variant="contained"
-                    >
-                      Create account
-                    </button>
-                  </Link>
-                </>
+                <></>
               </>
```

Comment:
We can write `null` instead 
`<>
<></>
</>`

Reply:
changed